src/commands/sarif/validation.ts

```diff
@@ -58,6 +58,18 @@ export const checkForError = (filePath: string): string[] => {
           }
         }
       }
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
That's not ideal, but the whole function does that so I'll let it pass. It could be better typed in the future 😁